### DISCUSSION TITLE
Is it free of cost or any premium charges?

### DISCUSSION THREAD

**Discussion by ebaraiya** [talsec/Free-RASP-Community#10] [Q&A] [answered]
** I have implemented this SDK in Android and got some emails from Talsec. I just want to know if is it free or has any premium cost.

**Comment by syakymchuk** [comment, marked as answer]
Hi @ebaraiya, 

freeRASP is a freemium product. It means it is free but it has important security limitations and Fair Usage Policies. It is designed to be a good start and basic security. For valuable commercial and especially FinTech apps, we strongly recommend considering commercial products. This [section ](https://github.com/talsec/Free-RASP-Community#bar_chart-enterprise-services)of readme provides the main diff between plans freeRASP and RASP+ [plans](https://github.com/talsec/Free-RASP-Community#bar_chart-enterprise-services) 

See more details about current and future free vs paid [differences here.](https://github.com/talsec/Free-RASP-Community/discussions/5)

You may also consider the upcoming [lightRASP plan](https://github.com/talsec/Free-RASP-Community/discussions/8). 

Happy to answer your questions or make a Demo on the call. Just pick a [time slot in Calendly](https://calendly.com/sergiy-yakymchuk-1/60min?month=2022-07).

Cheers, 
Sergiy Yakymchuk 
Talsec co-founder || OWASP member
